cg-spec/editors_draft.html

```diff
@@ -224,16 +227,16 @@ <h2>Background and Motivation</h2>
 
       <p>The RDF data model lets you state facts in three-part subject-predicate-object statements known as triples. For example, with a single RDF triple, you can say that employee38 has a familyName of "Smith". A triple's predicate is a property specified with an IRI (an Internationalized version of a URI), identifying that property in a globally unambiguous way. A triple's subject and object can each be an IRI referencing any entity, and the object can also be a literal value such as "Smith" or data of other types such as dates, numbers, or Boolean values.</p>
 
-      <p>Sometimes, we want the subject or object of a triple to refer to another triple. For example, the statement "according to employee22, employee38 has a jobTitle of 'Assistant Designer'" can be modeled as a triple with "according to" as its predicate, employee22 as its object, and another triple as its subject, namely the triple "employee38 has a jobTitle of 'Assistant Designer'". This use of a triple as the subject or object resource of another triple so that we can say things about that triple is known as <a href='https://www.w3.org/TR/rdf11-mt/#reification'>reification</a>.
-      The concept of reification has always been part of RDF, but expressing it in most RDF concrete syntaxes such as Turtle and N-Triples, as well as processing or querying it with SPARQL, has been verbose and cumbersome [[RDF-STAR-FOUNDATION]].</p>
+    <p>Sometimes, we want the subject or object of a triple to refer to another triple. For example, the statement "according to employee22, employee38 has a jobTitle of 'Assistant Designer'" can be modeled as a triple with "according to" as its predicate, employee22 as its object, and another triple as its subject, namely the triple "employee38 has a jobTitle of 'Assistant Designer'". This use of a triple as the subject or object resource of another triple so that we can say things about that triple is known as <a href='https://www.w3.org/TR/rdf11-mt/#reification'>reification</a>. The concept of reification has always been part of RDF, but expressing it in most RDF concrete syntaxes such as Turtle and N-Triples, as well as processing or querying it with SPARQL, has been verbose and cumbersome [[RDF-STAR-FOUNDATION]]. A notable exception is RDF/XML which provides the <a href="https://www.w3.org/TR/rdf-syntax-grammar/#section-Syntax-reifying">ID</a> attribute as syntactic sugar for the standard reification quadlet <span class=""heavy">[Note: see my <a href="https://github.com/w3c/rdf-star/pull/171#issuecomment-854648532">comment on issue #171</a> for why I think this sentence is important].</span></p>
```

Comment:
I don't see how this addition fits in a section called "background and motivation".

I am not denying that RDF/XML is better equipped than Turtle to handle standard reification, but this adds (or removes) nothing to the value of RDF-star, which addresses the problem in  a different way (namely, by extending the abstract syntax rather than just the concrete syntax).

As for the claim that RDF-star "requires extra effort" compared to standard reification, it kinds of contradicts everything that §1.2 aims to demonstrate. This is hardly a "motivation" for the rest of the work...

Finally, the mention of explainable AI, without any further explanation (!) is a bit cryptic...

Reply:
`rdf:ID` can not be said to be standard reification because it is restricted in several ways.

Reification "in the wild" often uses blank nodes for the subject of the reification. `rdf:ID` can not express this.

RDF/XML `rdf:ID` used on a property, always asserts the triple. 

RDF/XML `rdf:ID` used on a property or in `<rdf:Description>` has an implicit assumption of triple grouping by subject  (via the uniqueness rule) which Turtle does not.

RDF/XML `rdf:ID` can not be the object.

The RDF/XML document can not use `rdf:ID` on a property and also refer to it via `<rdf:Description>`. It violates the rule "_Each (rdf:ID attribute value, base URI) pair has to be unique in an RDF/XML document_".

This means the document has to use another way to annotate the triple, which means it must know the base URI in scope (`xml:base` can vary through the document). It also means it can occur in another document elsewhere (either in time or place).

Reply:
> I don't see how this addition fits in a section called "background and motivation".

But the other syntaxes mentioned do? 

> I am not denying that RDF/XML is better equipped than Turtle to handle standard reification, but this adds (or removes) nothing to the value of RDF-star, which addresses the problem in a different way (namely, by extending the abstract syntax rather than just the concrete syntax).

Both have their strengths and weaknesses. They are definitely operating in the same territory and it's good practice to mention related work and help the user make an informed decision. I find mentioning RDF/XML and its ID attribute especially fitting as this is not just a theoretical discussion but part of the use cases and discussions in calls with @JervenBolleman w.r.t. UniProt use case.

> As for the claim that RDF-star "requires extra effort" compared to standard reification, it kinds of contradicts everything that §1.2 aims to demonstrate. This is hardly a "motivation" for the rest of the work...

The sentence goes on hinting at new use cases that the proposed semantics enables, which AFAICT is indeed the motivation for the proposed semantics. So I think this sentence is really a worthwhile addition to this section. 
I did however make a mistake as "requires extra effort" relates to RDF standard reification which was not my intention. I meant to hint at the extra triples required in RDF-star when modelling occurrences or referentially transparent references. I'll have to re-word.

> Finally, the mention of explainable AI, without any further explanation (!) is a bit cryptic...

We could replace that mention by "the Superman problem", but that would be a lot less convincing. Honestly, I'm having trouble of coming up with any other example. But I also added an issue further below where I propose and argue that you add the  Explainable AI use case from your Lotico presentation. That would of course then be linked from here.